content/reference/user/private_locations/includes/control-plane.conf

```diff
@@ -0,0 +1,50 @@
+control-plane {
+  # Control plane token
+  token = "cpt_example_c7oze5djp3u14a5xqjanhcbqcw4sgptocfudrat3crauec3ugptggpjzg7tu1a3uco3uoa3sca4ur3mfgo5gka5dga5dqqboga3uoc3ige3scpjoctouccj18f1depy"
+  # Control plane description
+  description = "my control plane description"
+  # Locations configurations
+  locations = [
```

Comment:
It actually still is referred to as `pools` on dev

Reply:
https://github.com/gatling/frontline-cloud/pull/2425

Reply:
Yes, but there's an issue about it

Reply:
Yes, I'm on it

Reply:
https://github.com/gatling/frontline-cloud/pull/2425